templates/rancher.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>rancher</Name>
+  <Repository>rancher/rancher:latest</Repository>
+  <Registry>https://hub.docker.com/r/rancher/rancher</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://rancher.com/docs/rancher/v2.x/en/installation/other-installation-methods/single-node-docker/</Support>
+  <Project>https://github.com/rancher/rancher</Project>
+  <Overview>A container management platform built for organizations that deploy containers in production. The docker version of Rancher cannot be made HA (Highly Available), however the clusters it creates can be. It is also possible to connect to and use clusters created by rancher when it is down. Useful as a management interface for deploying, scaling, and editting apps on other systems. (in the cloud, a raspberry pi cluster, vms running k3os, etc.) Perfect for running and managing the apps that you'd rather not go down while maintaining your server.</Overview>
+  <Category/>
+  <WebUI>https://[IP]:[PORT:3443]</WebUI>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/rancher.png</Icon>
+  <Description>A container management platform built for organizations that deploy containers in production. The docker version of Rancher cannot be made HA (Highly Available), however the clusters it creates can be. It is also possible to connect to and use clusters created by rancher when it is down. Useful as a management interface for deploying, scaling, and editting apps on other systems. (in the cloud, a raspberry pi cluster, vms running k3os, etc.) Perfect for running and managing the apps that you'd rather not go down while maintaining your server.</Description>
+  <Networking>
+    <Mode>bridge</Mode>
+    <Publish>
+      <Port>
+        <HostPort>3080</HostPort>
+        <ContainerPort>80</ContainerPort>
+        <Protocol>tcp</Protocol>
+      </Port>
+      <Port>
+        <HostPort>3443</HostPort>
+        <ContainerPort>443</ContainerPort>
+        <Protocol>tcp</Protocol>
+      </Port>
+    </Publish>
+  </Networking>
+  <Data>
+    <Volume>
+      <HostDir>/mnt/user/appdata/rancher/</HostDir>
+      <ContainerDir>/var/lib/rancher</ContainerDir>
+      <Mode>rw</Mode>
+    </Volume>
+  </Data>
+  <Config Name="Web UI" Target="80" Default="80" Mode="tcp" Description="Container Port: 80" Type="Port" Display="always" Required="true" Mask="false">3080</Config>
+  <Config Name="Secure Web UI" Target="443" Default="443" Mode="tcp" Description="Container Port: 443" Type="Port" Display="always" Required="true" Mask="false">3443</Config>
+  <Config Name="Datastore" Target="/var/lib/rancher" Default="/mnt/user/appdata/rancher/" Mode="rw" Description="Container Path: /var/lib/rancher" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/rancher/</Config>
+</Container>
```

Comment:
Dont add "data" to the config, like the ports 3080 and 3443. Let the user choose the ports.
https://selfhosters.net/docker/templating/templating/#122-add-a-config

Reply:
Fixed in bcd9058daaf95da19ff9b08aed28c5b66f1c801d